src/economy.cpp

```diff
@@ -136,13 +133,62 @@ Money CalculateCompanyValue(const Company *c, bool including_loan)
 		}
 	}
 
+	return value;
+}
+
+/**
+ * Calculate the value of the company. That is the value of all
+ * assets (vehicles, stations) and money (including loan),
+ * except when including_loan is \c false which is useful when
+ * we want to calculate the value for bankruptcy.
+ * @param c the company to get the value of.
+ * @param including_loan include the loan in the company value.
+ * @return the value of the company.
+ */
+Money CalculateCompanyValue(const Company *c, bool including_loan)
+{
+	Money value = CalculateCompanyAssetValue(c);
+
 	/* Add real money value */
 	if (including_loan) value -= c->current_loan;
 	value += c->money;
 
 	return std::max<Money>(value, 1);
 }
 
+/**
+ * Calculate what you have to pay to take over a company.
+ *
+ * This is different from bankruptcy and company value, and involves a few
+ * more parameters to make it more realistic.
+ *
+ * You have to pay for:
+ * - The value of all the assets in the company.
+ * - The loan the company has (the investors really want their money back).
+ * - The profit for the next two years (if positive) based on the last year.
```

Comment:
```suggestion
 * - The profit for the next two years (if positive) based on the previous four quarters.
```
"last year" in itself is a bit confusing, lets assume today is June 6th 2023. Does it mean June 6th 2022 up to including June 5th 2023, or January 1st 2022 up to including December 31st 2023. But here it's actually like April 1st 2022 up to including March 31st 2022.